### PR TITLE
[SDK] Refactor hello world soak test

### PR DESCRIPTION
**Pull request by mbdcos**
`soak_secrets_update` and `soak_secrets_alive` are working. Also verified in a separate cluster with `secrets/hello-world` installed.

### CODE REVIEW

**Comment by mbdcos** [comment]
ping @susanxhuynh @benclarkwood

File: frameworks/helloworld/tests/test_soak.py [hunk 1]
```diff
@@ -65,27 +54,52 @@ def test_soak_secrets_update():
...
-    # revert back
+    # revert back to some other value
     cmd.run_cli("security secrets update --value=SECRET1 secrets/secret1")
     cmd.run_cli("security secrets update --value=SECRET2 secrets/secret2")
     cmd.run_cli("security secrets update --value=SECRET3 secrets/secret3")
+    test_soak_secrets_restart_hello0()
+
+
+@pytest.mark.soak_secrets_alive
+@dcos_1_10
```
Comment (benclarkwood) [inline frameworks/helloworld/tests/test_soak.py]: Nitpick: can you make this pytest.mark.dcos_1_10?

Comment (mbdcos) [inline frameworks/helloworld/tests/test_soak.py]: already pytest.mark

dcos_1_10 = pytest.mark.skipif('dcos_version_less_than("1.10")')
https://github.com/dcos/shakedown/blob/master/shakedown/dcos/cluster.py

May be using the following is easier:
@pytest.mark.skipif('shakedown.dcos_version_less_than("1.9")')

But documented way of doing is https://github.com/dcos/shakedown/blob/master/API.md#dcos_1_10

Comment (benclarkwood) [inline frameworks/helloworld/tests/test_soak.py]: Oh cool!

**Comment by benclarkwood** [comment]
The test failed because it couldn't find `dcos_1_10`

@kensipe are the shakedown docs inaccurate?

File: frameworks/helloworld/tests/test_soak.py (hunk 1 above)
Comment (mbdcos) [inline frameworks/helloworld/tests/test_soak.py]: switched to `@pytest.mark.skipif('shakedown.dcos_version_less_than("1.10")')`

**Comment by mbdcos** [comment]
for `@dcos_1_10`
use either 
   `from shakedown import *`
or 
  `from dcos.mesos import DCOSClient
  from distutils.version import LooseVersion
  import dcos
  import shakedown`

**Pull request merged**